lib/queries.js

```diff
@@ -183,4 +187,68 @@ export async function getAllOvoAndKboCouples() {
   return null;
 }
 
+export async function createRechtsVormStructure(){
+
+}
+
+export async function createNewKboOrg(kboOrg ,kboNummer){
+  const newKboOrgIdUuId = uuid();
+  const newKboIdUri = `http://data.lblod.info/id/kboBestuurseenheden/${newKboOrgIdUuId}`;
+
+  const contactPointUuidId = uuid();
+  const contactPointUri = `http://data.lblod.info/id/contact-punten/${contactPointUuidId}`;
+
+  let updateStr = `
+  INSERT {
+    GRAPH ?g {
+
+      ${sparqlEscapeUri(contactPointUri)} a <http://schema.org/ContactPoint> ;
+                                            <http://mu.semte.ch/vocabularies/core/uuid> ${sparqlEscapeString(contactPointUuidId)} ;
+                                            <http://schema.org/email> ${sparqlEscapeString(kboOrg.email)} ;
+                                            <http://schema.org/telephone> ${sparqlEscapeString(kboOrg.phone)} ;
+                                            <http://schema.org/contactType> ${sparqlEscapeString("Kbocontact")} ;
+                                            <http://xmlns.com/foaf/0.1/page> ${sparqlEscapeString(kboOrg.website)}.
+
+
+
+      ${sparqlEscapeUri(newKboIdUri)} a <http://mu.semte.ch/vocabularies/ext/KboEenheid> ;
+                                        <http://mu.semte.ch/vocabularies/core/uuid> ${sparqlEscapeString(newKboOrgIdUuId)} ;
+                                        <http://mu.semte.ch/vocabularies/ext/rechtsvorm> ${sparqlEscapeString(kboOrg.rechtsvorm)} ;
+                                        <http://www.w3.org/2004/02/skos/core#prefLabel> ${sparqlEscapeString(kboOrg.organisationName)} ;
+                                        <http://www.w3.org/2004/02/skos/core#altLabel> ${sparqlEscapeString(kboOrg.formalName)} ;
+                                        <https://data.lblod.info/vocabularies/generiek/geplandeEindDatum> ${sparqlEscapeString(kboOrg.changeTime)} ;
+                                        <http://schema.org/contactPoint> ${sparqlEscapeUri(contactPointUri)} .
+    }
```

Comment:
I would define the necessary prefixes to avoid repeating whole URIs throughout the queries. Imo this makes them more readable. As an example you can look at how it is done in another [service](https://github.com/lblod/construct-administrative-unit-relationships-service).